cmake/windows-setup.cmake

```diff
@@ -61,6 +61,7 @@ if(FLB_WINDOWS_DEFAULTS)
   set(FLB_IN_ELASTICSEARCH      Yes)
   # disable calyptia fleet management for now
   set(FLB_IN_CALYPTIA_FLEET     No)
+  set(FLB_IN_SPLUNK             Yes)
```

Comment:
It isn't necessary but other globally enabled plugins are explicitly enabled because this file is linked from the official doc:
https://docs.fluentbit.io/manual/installation/windows
For users, it is easy to understand which plugins are enabled for Windows.

Reply:
That's  a fair point. We might want to talk about it in the future but as far as this PR goes it's resolved.

Reply:
Yeah we discussed about parsing the output from `--help` or similar to auto-generate the docs with tags or a matricx for a plugin showing it is enabled/disable for Linux, Mac, Windows by default. Having to do it manually is not sustainable but as you say not for this PR.